src/components/__tests__/developerCard.test.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react"
+import { render } from "@testing-library/react"
+import DeveloperCard from "../DeveloperCard"
+
+describe("Test on DeveloperCard", () => {
+  const initialData = {
+    DeveloperName: "Jose Reyes Campusano",
+    SummaryDeveloper:
+      "Senior Front-end Developer | Web Designer | Hardcore Gamer | I am passionate about building scalable and appealing software",
+    Initials: "JR",
+    Image:
+      "https://upload.wikimedia.org/wikipedia/commons/3/3f/Juan_Perez_July_2013.jpg",
+    SkillsList: ["React", "Flutter", "C#", "Swift", "Kotlin"],
+    linkedin: "https://www.linkedin.com/in/linkeindTest/",
+    Twitter: "https://twitter.com/TwitterTest",
+  }
+
+  const card = render(
+    <DeveloperCard
+      developerName={initialData.DeveloperName}
+      initials={initialData.Initials}
+      image={initialData.Image}
+      skillsList={initialData.SkillsList}
+      summaryDeveloper={initialData.SummaryDeveloper}
+      linkedin={initialData.linkedin}
+      twitter={initialData.Twitter}
+    />
+  )
+
+  it("renders card with initial data", () => {
+    expect(card).toMatchSnapshot()
```

Comment:
```suggestion
    expect(card.toJSON()).toMatchSnapshot()
```
Uno solo toma el contenido del component, no todo el objeto de testing.